File: integration_mob/src/com/mycompany/Entite/Reclamation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.Entite;

import java.util.Date;

/**
 *
 * @author wildchild
 */
public class Reclamation {

    private int id_pub = -1;
    private int id_usr;
    private String sujet;
    private Date datePub;
    private String description;
    private String type;
    private String etat;

    public String getEtat() {
        return etat;
    }

    // a list maybe
    public void setEtat(String etat) {
        this.etat = etat;
    }

    private int use_id_usr = -1;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Reclamation(int id_pub, int id_usr, int use_id_usr, Date datePub, String sujet, String description) {
        this.id_pub = id_pub;
        this.id_usr = id_usr;
        this.sujet = sujet;
        this.datePub = datePub;
        this.description = description;
        this.use_id_usr = use_id_usr;
    }

    public Reclamation() {

    }

    public int getId_pub() {
        return id_pub;
    }

    public void setId_pub(int id_pub) {
        this.id_pub = id_pub;
    }

    public int getId_usr() {
        return id_usr;
    }

    public void setId_usr(int id_usr) {
        this.id_usr = id_usr;
    }

    public String getSujet() {
        return sujet;
    }

    public void setSujet(String sujet) {
        this.sujet = sujet;
    }

    public Date getDatePub() {
        return datePub;
    }

    public void setDatePub(Date datePub) {
        this.datePub = datePub;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getUse_id_usr() {
        return use_id_usr;
    }

    public void setUse_id_usr(int use_id_usr) {
        this.use_id_usr = use_id_usr;
    }

    @Override
    public String toString() {
        return "Reclamation{" + "id_pub=" + id_pub + ", id_usr=" + id_usr + ", sujet=" + sujet + ", datePub=" + datePub + ", description=" + description + ", target user=" + use_id_usr + '}';
    }

}

File: piproj/src/ReclamationBundle/Entity/Reclamation.php
<?php

namespace ReclamationBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reclamation
 *
 * @ORM\Table(name="reclamation", indexes={@ORM\Index(name="FK_CONCERNE", columns={"ID_USR"})})
 * @ORM\Entity
 */
class Reclamation
{
    /**
     * @var integer
     *
     * @ORM\Column(name="ID_PUB", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var \AdminUserBundle\Entity\User
     *
     * @ORM\ManyToOne(targetEntity="AdminUserBundle\Entity\User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="USE_ID_USR", referencedColumnName="id")
     * })
     */
    private $useIdUsr;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DATEPUB", type="date", nullable=true)
     */
    private $datepub;

    /**
     * @var string
     *
     * @ORM\Column(name="SUJET_REC", type="string", length=100, nullable=true)
     */
    private $sujetRec;

    /**
     * @var string
     *
     * @ORM\Column(name="DESCRIPTION", type="text", length=65535, nullable=true)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="ETAT", type="string", length=20, nullable=true)
     */
    private $etat;

    /**
     * @var string
     *
     * @ORM\Column(name="TYPE_REC", type="string", length=50, nullable=true)
     */
    private $typeRec;

    /**
     * @var \AdminUserBundle\Entity\User
     *
     * @ORM\ManyToOne(targetEntity="AdminUserBundle\Entity\User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="ID_USR", referencedColumnName="id")
     * })
     */
    private $idUsr;


    /**
     * Get id
     *
     * @return integer
     */
    public function getId()
    {
        return $this->id;
    }


    /**
     * Set datepub
     *
     * @param \DateTime $datepub
     *
     * @return Reclamation
     */
    public function setDatepub($datepub)
    {
        $this->datepub = $datepub;

        return $this;
    }

    /**
     * Get datepub
     *
     * @return \DateTime
     */
    public function getDatepub()
    {
        return $this->datepub;
    }

    /**
     * Set sujetRec
     *
     * @param string $sujetRec
     *
     * @return Reclamation
     */
    public function setSujetRec($sujetRec)
    {
        $this->sujetRec = $sujetRec;

        return $this;
    }

    /**
     * Get sujetRec
     *
     * @return string
     */
    public function getSujetRec()
    {
        return $this->sujetRec;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Reclamation
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set etat
     *
     * @param string $etat
     *
     * @return Reclamation
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * Get etat
     *
     * @return string
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * Set typeRec
     *
     * @param string $typeRec
     *
     * @return Reclamation
     */
    public function setTypeRec($typeRec)
    {
        $this->typeRec = $typeRec;

        return $this;
    }

    /**
     * Get typeRec
     *
     * @return string
     */
    public function getTypeRec()
    {
        return $this->typeRec;
    }

    /**
     * Set idUsr
     *
     * @param \AdminUserBundle\Entity\User $idUsr
     *
     * @return Reclamation
     */
    public function setIdUsr(\AdminUserBundle\Entity\User $idUsr = null)
    {
        $this->idUsr = $idUsr;

        return $this;
    }


    /**
     * Get idUsr
     *
     * @return \AdminUserBundle\Entity\User
     */
    public function getIdUsr()
    {
        return $this->idUsr;
    }

    /**
     * Set useIdUsr
     *
     * @param \AdminUserBundle\Entity\User $useIdUsr
     *
     * @return Reclamation
     */
    public function setUseIdUsr(\AdminUserBundle\Entity\User $useIdUsr = null)
    {
        $this->useIdUsr = $useIdUsr;

        return $this;
    }

    /**
     * Get useIdUsr
     *
     * @return \AdminUserBundle\Entity\User
     */
    public function getUseIdUsr()
    {
        return $this->useIdUsr;
    }
}
